src/styles/shared/_accessibility.scss

```diff
@@ -1,16 +1,16 @@
 /// Used to hide an element visually, but keeping it accessible for
 /// accessibility tools.
 
+/// styles referenced from GOV.UK design system
+/// https://github.com/h5bp/main.css/issues/12#issuecomment-451965809
 @mixin visually-hidden {
-  // Need to make sure we override any existing styles.
-  // stylelint-disable declaration-no-important
-  position: absolute !important;
-  top: 0;
-  clip: rect(1px, 1px, 1px, 1px) !important;
-  overflow: hidden !important;
-  height: 1px !important;
-  width: 1px !important;
-  padding: 0 !important;
-  border: 0 !important;
-  // stylelint-enable declaration-no-important
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  clip-path: inset(50%);
+  border: 0;
+  white-space: nowrap;
```

Comment:
Looks great, we just need the `!important` added back as removing that override could create a big risk.

Reply:
sounds good, should I add it back on all of the same attributes?

Reply:
👍 I think we'll need !important on all of the properties here because we can't guarantee that consumers won't have some other rule that overrides the mixin.

We should add the comment back as well:
`// Need to make sure we override any existing styles.`

Reply:
This looks great! Thanks all!